homework/index.js

```diff
@@ -29,25 +32,150 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  // 1- create function to add row
+  function createRow(table, label, content) {
+    const row = createAndAppend('tr', table);
+    createAndAppend('th', row, { text: label });
+    createAndAppend('td', row, { text: content });
+    return row;
+  }
+
+  // 2- create details of repositories
+  function renderRepoDetails(detailsSection, repo) {
+    const li = createAndAppend('table', detailsSection, { class: 'item' });
+    const table = createAndAppend('table', li);
+    // 2-1 add repository name
+    const repoName = createRow(table, 'Repository:', '');
+    createAndAppend('a', repoName.lastChild, {
+      href: repo.html_url,
+      text: repo.name,
+      target: '_blank',
+    });
+    // 2-2 add description
+    if (repo.description) {
+      createRow(table, 'Description:', repo.description);
+    } else {
+      createRow(table, 'Description:', 'N/A');
+    }
```

Comment:
Instead of the `if` statement in lines 55-59 you can use a _short-circuit_ expression like this:

```js
createRow(table, 'Description:', repo.description || 'N/A');
```

I already gave this feedback in week 1 but somehow you seem to have missed it.